### PR TITLE
Arshamian 2022 added

### PR DESCRIPTION
**Pull request by TGouldBates**

### CODE REVIEW

**Comment by rgerkin** [comment]
@TGouldBates Looks like this PR still has the files from Mayhew as well.  Can you de-stage them from this PR?

File: arshamian_2022/manifest.toml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Manifest for Arshamian et al 2022
+[source]
+doi = "10.1101/2021.03.01.433367"
+title = "The perception of odor pleasantness is shared across cultures."
+authors = "A. Arshamian, R.C. Gerkin, N. Kruspe, E. Wnuk, S. Floyd, C. O'Meara, G. Garrido Rodriguez, J. Lundstrom, J.D. Mainland, A. Majid"
+extra = ""
+
+[raw]
+data/All data intensity.txt = "Ordor intensity rankings from global participants"
```
Comment (rgerkin) [inline arshamian_2022/manifest.toml:9]: @TGouldBates Extremely minor: the .toml file will not correctly parse if the keys have any spaces in them.  I'll handle this (it's a quick edit for me) but in the future go ahead and change file names to add underscores so there are no spaces.

Comment (TGouldBates) [reply arshamian_2022/manifest.toml:9]: Got it, no problem.

**Comment by TGouldBates** [comment]
No problem.

I removed the Mayhew directory from my Arshamian branch. Did that percolate
through to the PR so that you only see the Arshamian directory in that PR?

File: arshamian_2022/behavior_2.csv [hunk 2]
```diff
@@ -0,0 +1,126 @@
+ParticipantID,1) Odor intensity (most intense),2) Odor intensity,3) Odor intensity,4) Odor intensity,5) Odor intensity,6) Odor intensity,7) Odor intensity,8) Odor intensity,9) Odor intensity,10) Odor intensity (least intense/weakest)
```
Comment (rgerkin) [inline arshamian_2022/behavior_2.csv:1]: - Can you format this file to use the CIDs.  Then molecules.csv can be used to decode the rest.
- Can you retitle the columns to just be the ranks 1-10, or "Rank 1", "Rank 2", etc.?  If you think it is unclear the .toml file can be used to give more description.  The goal in these processed files is to retain the information but in a compact form, not necessarily to retain the exact text used by the original data collectors.

File: arshamian_2022/behavior_1.csv [hunk 3]
```diff
@@ -0,0 +1,2831 @@
+CID,ParticipantID,Ranking,Subsistence
```
Comment (rgerkin) [inline arshamian_2022/behavior_1.csv:1]: - Subsistence is a property of the participant and should go in the subjects.csv and not in behavior files.

File: arshamian_2022/behavior_3.csv [hunk 4]
```diff
@@ -0,0 +1,101 @@
+CID,RaterID,Rep1,Rep2
```
Comment (rgerkin) [inline arshamian_2022/behavior_3.csv:1]: The raters here are still subjects (for a different experiment) so give them ParticipantIDs in subjects.csv and then use those IDs here.  You can leave their subsistence column blank, and their group can be "Philadelphia".

**Review by rgerkin — CHANGES_REQUESTED** [review]

**Comment by TGouldBates** [comment]
Got it. Changes implemented and pushed.

**Pull request closed without merging**